src/styles/index.scss

```diff
@@ -1,3 +1,39 @@
-body {
+@import './utils/variables';
+@import './utils/cycles';
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
+
+.calendar {
+  gap: $gap;
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(7 * $cell-size + 6 * $gap);
```

Comment:
It is recommended to use a variable for the number of days in a week (7) to make the calculation more understandable and maintainable.